package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "nucypher-contracts",
+  "version": "0.0.1",
+  "license": "GPL-3.0-or-later"
```

Comment:
Do we want to have AGPL here?

Reply:
I think so, our source files specify `// SPDX-License-Identifier: AGPL-3.0-or-later`

Reply:
Threshold-network/solidity-contracts also have this License Identifier.

https://github.com/threshold-network/solidity-contracts/blob/main/package.json